server/tests/lib/quotes.test.ts

```diff
@@ -33,4 +33,10 @@ describe("Quotes", () => {
       expect(Quotes.getRandom()).to.equal(Quotes.quotes[randomizedIndex]);
     });
   });
+
+  describe("getGroup", async () => {
+    it("should return groups of quotes", async () => {
+      expect(Quotes.getGroup()).to.equal(Quotes.quotes);
```

Comment:
This is failing because `getGroup()` does not actually equall the static value `.quotes` from Quotes. Instead what you can do is iterate through the output of Quotes.getGroup() and use includes matcher - https://www.chaijs.com/api/bdd/#method_include.

```
// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
Quotes.getGroup().forEach((group) => {
    expect(Quotes.quotes).to.include(group);
})
```
^ this way we are checking that each group we make has values from the `Quotes.quotes` data set.